ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -547,11 +552,10 @@ Set Up Project
     ...                creates caikit runtime. This can be used as test setup
     [Arguments]    ${namespace}    ${single_prj}=${TRUE}    ${enable_metrics}=${FALSE}    ${dc_name}=kserve-connection
     IF    ${single_prj}
-        Clean All Models Of Current User
+        Run Keyword And Continue On Failure    Clean All Models Of Current User
     END
     Open Data Science Projects Home Page
-    Wait For RHODS Dashboard To Load    wait_for_cards=${FALSE}    expected_page=Data Science Projects
```

Comment:
why?

Reply:
Because that's what the KW docstring says:

> Set Up Project
>     [Documentation]    Creates the DS Project (if not exists)

Reply:
> Because Open Data Science Projects Home Page already does it :)

Ok

I wasn't questioning the create DS project part :)